src/lib/components/Tag/index.tsx

```diff
@@ -0,0 +1,21 @@
+import classNames from 'classnames';
+import { TagProps } from './type';
```

Comment:
@ryu9663 
준열님 요거 혹시 
```suggestion
import { TagProps } from './type';
import type {TagProps} from './type'
```

으로 하시는건 어떤가요? 린트룰로 정해도 될것 같습니다

Reply:
린트에 한번 넣어볼게요!